java/wavefrontsender.md

```diff
@@ -0,0 +1,212 @@
+# WavefrontSender
+
+## Table of Content
+
+* [Overview](#Overview)
+* [Option 1: Sending Data via the Wavefront Proxy](#Option-1-Sending-Data-via-the-Wavefront-Proxy)
+* [Option 2: Sending Data via Direct ingestion](#Option-2-Sending-Data-via-Direct-ingestion)
+* [Share a WavefrontSender](#Share-a-WavefrontSender)
+* [What's next](#What's-next)
+
+## Overview
+
+You can choose to send metrics, histograms, or trace data from your application to the Wavefront service using one of the following techniques:
+
+* Use a [**Wavefront proxy**](https://docs.wavefront.com/proxies.html), which then forwards the data to the Wavefront service. This is the recommended choice for a large-scale deployment that needs resilience to internet outages, control over data queuing and filtering, and more.
+* Use [**direct ingestion**](https://docs.wavefront.com/direct_ingestion.html) to send the data directly to the Wavefront service. This is the simplest way to get up and running quickly. 
+
+The `WavefrontSender` interface has two implementations. Instantiate the implementation that corresponds to your choice:
```

Comment:
Could probably omit "Instantiate the implementation that corresponds to your choice:"